upgrade/upgradetest/fork.go

```diff
@@ -19,6 +19,7 @@ const (
 	Etna
 	Fortuna
 	Granite
+	Helicon // naming credit to @bernard-avalabs!
```

Comment:
Not trying to take away any credit... But it looks weird in the code
```suggestion
	Helicon
```

Reply:
I'll remember @bernard-avalabs... I'll remember...

<img width="612" height="408" alt="image" src="https://github.com/user-attachments/assets/3b686b84-3f58-48ee-aa82-b1adf661cffd" />